src/test/java/uk/gov/justice/digital/hmpps/vsip/cucumber/steps/ViewVisitsByDateStep.java

```diff
@@ -47,4 +58,29 @@ public void iSeeOnViewVisitsByDatePage(String errMsg) {
         viewVisitsByDatePage.checkErrorMessage();
     }
 
+    @And("I capture date the Booking is booked")
+    public void iCaptureDateTheBookingIsBooked() {
+        //capturing date from booking confirmation page
+        String originalDateValue = confirmationPage.getDateBooked();
+        System.out.println("Original Date value from confirmation page: " + originalDateValue);
+
+        testContextService.setOriginalDateBookingReference(originalDateValue);
+        System.out.println("Original Date Booking Reference: " + testContextService.getOriginalDateBookingReference());
+
+        //formatting the date to pass to API
+        DateTimeFormatter originalFormatter = DateTimeFormatter.ofPattern("EEEE d MMMM yyyy");
+        // Parse the original date string
+        LocalDate originalDate = LocalDate.parse(originalDateValue, originalFormatter);
+        // Define the formatter for the desired output format
+        DateTimeFormatter outputFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+        // Format the date to the desired output format
+        String formattedDate = originalDate.format(outputFormatter);
```

Comment:
should we move these to a getFormattedDate method?

Reply:
Not my code will need to be fixed up later, lots more work to be done